src/main/java/com/stock/stocklist/controller/StockListController.java

```diff
@@ -55,4 +56,11 @@ public ResponseEntity<MessageResponse> update(@PathVariable Integer id, @Request
         return ResponseEntity.ok(message);
     }
 
+    @DeleteMapping("/stockList/{id}")
+    public ResponseEntity<MessageResponse> delete(@PathVariable Integer id) {
+        stockListService.delete(id);
+        MessageResponse message = new MessageResponse("data Deleted");
```

Comment:
今更気づいたのですが、
```suggestion
        MessageResponse message = new MessageResponse("data deleted");
```
のようにせずにDeletedにしたのって何か理由はありますか？
Updatedなどほかも同じくですね。

Reply:
c16cf5b2c033f6f0aaef90cadece17fc71354bd6
小文字に統一する形で修正しました。
ご確認お願いいたします。